### ISSUE TITLE
Pass tuples for size in simple Python interface

### ISSUE THREAD

**Issue by janden** [flatironinstitute/finufft#112]
Right now, we have to pass multiple arguments for the size in type 1, which means we have calls like
```
f = finufftpy.nufft2d1(x, y, c, N, N)
```
A more natural call would be
```
f = finufftpy.nufft2d1(x, y, c, (N, N))
```
which would be more in line with the plan interface, would allow the user to do things like `finufftpy.nufft2d1(x, y, c, f.shape)`, and provides for a simpler calling sequence when the grid is square:
```
f = finufftpy.nufft2d1(x, y, c, N)
```

**Comment by ahbarnett** [comment]
I agree. This matches guru in C++. I may change the matlab to match this too.

**Comment by lu1and10** [comment]
fixed in commit 6b060418dc5b4dc1f9dfe6f8398418e1cc36b82e of branch gi_v2_nthr, @janden you may want to double check.

**Comment by janden** [comment]
Looks good. Thanks!
